prody/proteins/interactions.py

```diff
@@ -0,0 +1,2032 @@
+            'pdbfixer': PDBFixer and OpenMM
+    :type method: str, by default is 'openbabel'
+    
+    :arg pH: pH value applyed only for PDBfixer.
+    :type pH: int or float
+    
+    Instalation of Openbabel:
+    conda install -c conda-forge openbabel    
+
+    Find more information here: https://anaconda.org/conda-forge/openbabel
+                                https://github.com/openmm/pdbfixer
+    Program will create new file in the same directory with 'addH_' prefix.
+    
+    .. [NO11] O'Boyle, N. M., Banck M., James C. A., Morley C., Vandermeersch T., Hutchison G. R. 
+    Open Babel: An open chemical toolbox *Journal of cheminformatics* **2011** 3:1-14. """
+    
+    if method == 'openbabel':
+        try:
+            #import openbabel
+            from openbabel import openbabel
+            obconversion = openbabel.OBConversion()
+            obconversion.SetInFormat("pdb")
+            mol = openbabel.OBMol()
+            obconversion.ReadFile(mol, pdb)
+            mol.AddHydrogens()
+            obconversion.WriteFile(mol, 'addH_'+pdb)
+            LOGGER.info("Hydrogens were added to the structure. Structure {0} is saved in the local directry.".format('addH_'+pdb))
+        except ImportError:
+            raise ImportError("Install Openbabel to add hydrogens to the structure or use PDBFixer/OpenMM.")
+            
+    elif method == 'pdbfixer':
+        try:
+            from pdbfixer import PDBFixer
+            try:
+                import openmm
+                from openmm.app import PDBFile
+            except ImportError:
+                from simtk.openmm.app import PDBFile
+            
+            fixer = PDBFixer(filename=pdb)
+            fixer.findMissingResidues()
+            fixer.removeHeterogens(True)
+            fixer.findMissingAtoms()
+            fixer.addMissingAtoms()
+            fixer.addMissingHydrogens(pH)
+            PDBFile.writeFile(fixer.topology, fixer.positions, open('addH_'+pdb, 'w'))
+            LOGGER.info("Hydrogens were added to the structure. Structure {0} is saved in the local directry.".format('addH_'+pdb))
+
+        except ImportError:
+            raise ImportError('Install PDBFixer and OpenMM in order to fix the protein structure.')
+
+    else:
+        raise TypeError('Method should be openbabel or pdbfixer')
+    
+    
+def calcHydrogenBonds(atoms, distA=3.0, angle=40, cutoff_dist=20, **kwargs):
+    """Compute hydrogen bonds for proteins and other molecules.
+    
+    :arg atoms: an Atomic object from which residues are selected
+    :type atoms: :class:`.Atomic`
+    
+    :arg distA: non-zero value, maximal distance between donor and acceptor.
+    :type distA: int or float, default is 3.0
+    
+    :arg angle: non-zero value, maximal (180 - D-H-A angle) (donor, hydrogen, acceptor).
+    :type distA: int or float, default is 40.
+    
+    :arg cutoff_dist: non-zero value, interactions will be found between atoms with index differences
```

Comment:
I fixed this here. Please apply to calcChHydrogenBonds or better still remove it from there and say that kwargs are the same ones